src/views/Voting/helpers.ts

```diff
@@ -181,34 +186,20 @@ function calculateVotingPower(scoresList: GetScoresResponse, voters: string[], s
 
     const cakePoolBalance = new BigNumber(userStakeInCakePools[address])
     // calculate cakeBnbLpBalance
-    const totalSupplyLP = new BigNumber(cakeBnbLpTotalSupplies[address])
-    const cakeBnbLpReserve0 = new BigNumber(cakeBnbLpReserve0s[address])
-    const cakeBnbLpCakeBnbBalance = new BigNumber(cakeBnbLpCakeBnbBalances[address])
-    const cakeBnbLpBalance = cakeBnbLpCakeBnbBalance.times(cakeBnbLpReserve0).div(totalSupplyLP)
-
-    // calculate poolsBalance
-    const poolStartIndex = scoresListIndex.poolStart
-    let poolsBalance = new BigNumber(0)
-    for (let i = poolStartIndex; i < scoresList.length; i++) {
-      const currentPoolBalance = new BigNumber(scoresList[i][address])
-      poolsBalance = poolsBalance.plus(currentPoolBalance)
-    }
+    const cakeBnbLpBalance = cakeBnbLpBalances[address]
 
     const total = cakeBalance
       .plus(cakeVaultBalance)
       .plus(cakePoolBalance)
       .plus(IFOPoolBalance)
-      .plus(cakeBnbLpBalance)
-      .plus(poolsBalance)
-      .div(TEN_POW_18)
-      .toFixed(18)
+      .plus(new BigNumber(cakeBnbLpBalance).times(TEN_POW_18))
```

Comment:
I think we should keep the div and tofixed here

Reply:
calculation on below
https://github.com/pancakeswap/pancake-frontend/blob/ce75a69b31bb369cc683141cfdbafbca474f237a/src/views/Voting/helpers.ts#L284-L294

Reply:
👍